lib/matplotlib/cbook.py

```diff
@@ -2244,6 +2244,21 @@ def _reshape_2D(X):
     return X
 
 
+def ensure_3d(arr):
```

Comment:
I would have thought that an underscore in the beginning means a function that is only called from within cbook, but clearly some underscored functions are called from elsewhere. What is the intended meaning?